File: moban_java/src/main/java/org/jdk/project/dto/config/LoungeUpsertRequest.java
package org.jdk.project.dto.config;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class LoungeUpsertRequest {
  private String roomCode;
  private String deptName;
  private String terminal;
  private String locationDesc;
  private Boolean enabled;
  private String remark;
}

File: backend/src/main/kotlin/org/jdk/project/dto/config/LoungeUpsertRequest.kt
package org.jdk.project.dto.config

data class LoungeUpsertRequest(
    var roomCode: String? = null,
    var deptName: String? = null,
    var terminal: String? = null,
    var locationDesc: String? = null,
    var enabled: Boolean? = null,
    var remark: String? = null,
)
